src/unix/uilocale.cpp

```diff
@@ -87,6 +87,7 @@ class wxUILocaleImplUnix : public wxUILocaleImpl
 
     wxLocaleIdent m_locId;
     wxString m_codeset;
+    wxString m_nameDefault;
```

Comment:
Why isn't this just called `m_name`? There doesn't seem to be anything particular "default" about it, it's used for any locale, not just the default one.

Reply:
Well, the member variable is only set and used for the default user locale. Otherwise it is left blank, and `m_locId.GetName()` will be used to retrieve the name of the locale. I can use `m_name`, of course, but `m_nameDefaultLocale` may be even better.

Reply:
But the ctor does
```cpp
m_nameDefault = m_locId.GetName();
```
unconditionally currently. It's true that later we only use it if `m_locId.GetName().empty()` but I don't really understand why don't we just always use it -- as long as we already have it anyhow.

Reply:
Yes, most likely we can change method `GetName` to always simply return `m_name`.